plugin-root/settings.gradle.kts

```diff
@@ -0,0 +1,17 @@
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        mavenCentral()
+        jcenter()
+        google()
+    }
+    resolutionStrategy {
+        eachPlugin {
+            if ("com.android" in requested.id.id) {
+                useModule("com.android.tools.build:gradle:${requested.version}")
+            }
+        }
+    }
```

Comment:
This is not needed anymore. This `plugin` project doesn't have any `com.android` deps.